src/services/PlacesService.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+const env = import.meta.env;
+const apiKey = env.VITE_GMAPS_API_KEY;
+const baseUrl = 'https://maps.googleapis.com/maps/api/place/';
+
+class PlacesService {
```

Comment:
Why do we using class notation here? We might export individual functions instead. Do we need to have some state inside service?

Reply:
Using class notation here allows to encapsulate related functionality and potentially manage data in the future, especially if service will be extended later. Exporting individual functions works too, but a class provides a cleaner structure for grouping and maintaining related methods

Reply:
What about state, I think all of state and state management should be in corresponding components and state managers, so class it is just construction that unite related methods